docs/index.html

```diff
@@ -33,20 +33,20 @@
     <nav>
         <div class="nav-wrapper">
             <a href="./index.html" class="brand-logo"><img src="resources/logo240.png" class="logo-img" /><span
-                    class="hide-on-small-only logo-text">emeInvestor bot</span></a>
+                    class="hide-on-small-only logo-text">emeInvestor Bot</span></a>
             <a data-target="mobile-sidebar" class="sidenav-trigger"><i class="material-icons">menu</i></a>
 
             <ul id="nav-mobile" class="right hide-on-med-and-down">
-                <li>
+                <!-- <li>
```

Comment:
Hide the search bars, as they don't work anyways. Better UX and UI to just remove them until functional.